index.html

```diff
@@ -42,17 +50,17 @@
 
 We consider privacy a core design principle and differentiator for the web platform (see: [Ethical Web Principles](https://www.w3.org/TR/ethical-web-principles/#privacy), [Unsanctioned Web Tracking](https://www.w3.org/2001/tag/doc/unsanctioned-tracking/), [Private Browsing Modes](https://www.w3.org/2001/tag/doc/private-browsing-modes/), [Privacy Principles](https://www.w3.org/TR/privacy-principles/), [Security & Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)).
 
-Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/), [Chrome](https://blog.google/products/chrome/updated-timeline-privacy-sandbox-milestones/)).  The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
+Many browsers are restricting, deprecating, or completely removing support for third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```

Comment:
Given the title change, maybe we should change "The TAG supports efforts to drop support for third-party cookies" to something more direct.

```suggestion
Many browsers are restricting, deprecating, or completely removing support for third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG insists that all browsers drop support for third-party cookies, as this improves the privacy preserving features of the web platform.
```

Reply:
```suggestion
Many browsers have restricted and/or removed support for third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG calls for all browsers to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```